Update CONTRIBUTING.md (#107)
elastic/ems-landing-page: README.md

## Continuous Integration and Deployment
* When a PR is merged Jenkins will run `deployStaging.sh` script, which will place code into the staging bucket.

> * Deploying to production requires manually triggering [this Jenkins job](https://kibana-ci.elastic.co/job/elastic+ems-landing-page+deploy/) to run the `deployProduction.sh` script. This will rsync files of the chosen branch from the staging bucket to the production bucket. To trigger, log in and click the "Build with Parameters" link.

Reviewer: ```suggestion
* Deploying to production requires manually triggering [this Jenkins job](https://kibana-ci.elastic.co/job/elastic+ems-landing-page+deploy/) to run the [deployProduction.sh](deployProduction.sh) script. This will rsync files in all branches to the production bucket. To trigger, log in and click the "Build with Parameters" link.
```
